src/control-plane/backend/lambda/api/test/api/pipeline.test.ts

```diff
@@ -1114,7 +1156,8 @@ describe('Pipeline test', () => {
           },
         ],
         templateInfo: {
-          isLatest: false,
+          isLatest: true,
+          pipelineVersion: 'v1.0.0',
```

Comment:
What's the difference between pipeline version and solution version?

How do we assign then use `isLatest` flag?

Reply:
solution version is the version that save in DDB table.  pipeline version is a attribute of pipeline that set when create/upgrade.

If **isLatest === false** frontend will show Upgrade button.

Reply:
So is solution version always the version of web console? is it a volatile one? do you fetch it from the metadata table in runtime?

Reply:
solution version fetch from metadata table in runtime.